Mathlib/LinearAlgebra/Dimension/ErdosKaplansky.lean

```diff
@@ -0,0 +1,144 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Johannes Hölzl, Sander Dahmen,
+Kim Morrison, Chris Hughes, Anne Baanen, Junyan Xu
+-/
```

Comment:
The content of this file was added completely by me in [#9159](https://github.com/leanprover-community/mathlib4/pull/9159/files#diff-cb173017e1157ddc4b9f868be06c18ec2a38f8cf82e856e18d59ed49f97395d8R150)
Is there any automation at all to keep authorship record straight in splitting PRs?